### ISSUE TITLE
AUTO DJ MIXXX 2.4

### ISSUE THREAD

**Issue by gesacyprus** [mixxxdj/mixxx#12835]
### Bug Description

when I start auto dj I receive the following error message
"Mixxx auto dj requires two decks assigned to opposite sides of the crossfader"

### Version

2.4 new release

### OS

Windows 10

**Comment by Swiftb0y** [comment]
yes, that's intentionally. what skin do you use? Have you made sure you actually assigned two decks to opposite sites of the crossfader?

**Comment by zebebop** [comment]
It's not a critical thing. When I first started version 2.4, the two decks were on the same side. Just reassign them and you're done. It may just be necessary to correct a few lines in the code so that the 2 decks are in opposition at the first start.

**Comment by ronso0** [comment]
> It may just be necessary to correct a few lines in the code so that the 2 decks are in opposition at the first start

That is the case already. I assume you changed the xfader assignment and didn't notice until AutoDJ complained about it.

**Comment by ronso0** [comment]
FYI xfader assignment is persistent now, i.e. it save across restarts, if you changed it once it'll stay like that.

**Comment by gesacyprus** [comment]
You are Absolutely right, I reestablished the cross faders to the correct position and all working fine Thank you

 �

**Comment by gesacyprus** [comment]
Thanks for the tip 

All working fine

 �

**Comment by gesacyprus** [comment]
Hello, hope all is well 

I encountered an issue which in my opinion need to be fixed

It was easy to export a playlist to ITunes in Pls, and M3U formats, with the 2.5.0 version is not. possible any more.

Furthermore I tried to drag few songs from MIXXX to ITunes it also did not work

Best regards 

Georges Samaha 

 �

From: Georges Samaha ***@***.***> 
Sent: Monday, February 19, 2024 12:17 PM
To: 'mixxxdj/mixxx' ***@***.***>
Subject: RE: [mixxxdj/mixxx] AUTO DJ MIXXX 2.4 (Issue #12835)

 �

Thanks for the tip 

All working fine

 �

**Comment by ronso0** [comment]
@gesacyprus this is an entirely unrelated issue, please file a new bug report (if you can reproduce it with Mixxx 2.5)
